pages/404.tsx

```diff
@@ -0,0 +1,23 @@
+import SignDirectionIcon from 'mdi-react/SignDirectionIcon'
+import * as React from 'react'
+import Layout from '../components/Layout'
+
+export class NotFoundPage extends React.Component<any, {}> {
+    public render(): JSX.Element {
+        return (
+            <Layout location={this.props.location} className="bg-white">
+                <div className="container center error-page text-dark">
+                    <div className="error-page__circle rounded-circle">
+                        <div className="error-page__icon">
+                            <SignDirectionIcon />
+                        </div>
+                    </div>
+                    <h1>404: Not Found</h1>
+                    <p>Sorry, the requested URL was not found.</p>
+                </div>
```

Comment:
Recommended change:

                    <h1>404: Page Not Found</h1>
                    <p>The requested URL was not found, return to <a href="https://learn.sourcegraph.com">Sourcegraph Learn</p>

Reply:
The second line could be broken up to two lines I think.

I think we don't need to apologize and I think including the link to the homepage would help the user recover.